snuba/datasets/processors/search_issues_processor.py

```diff
@@ -99,6 +127,43 @@ def _process_user(
 
         return
 
+    def _process_tags(
+        self, event_data: IssueEventData, processed: MutableMapping[str, Any]
+    ) -> None:
+        tags_maybe = event_data.get("tags", None)
+        if not tags_maybe:
+            processed["tags.key"], processed["tags.value"] = [], []
+        tags: Mapping[str, Any] = _as_dict_safe(cast(Dict[str, Any], tags_maybe))
+        processed["tags.key"], processed["tags.value"] = extract_extra_tags(tags)
+        promoted_tags = {col: tags[col] for col in self.PROMOTED_TAGS if col in tags}
+        processed["release"] = promoted_tags.get(
+            "sentry:release",
+            event_data.get("release"),
+        )
+        processed["environment"] = promoted_tags.get(
+            "environment", event_data.get("environment")
+        )
+        processed["user"] = promoted_tags.get("sentry:user")
+        processed["dist"] = _unicodify(
+            promoted_tags.get("sentry:dist", event_data.get("dist")),
+        )
+
+    def _process_request_data(
+        self, event_data: IssueEventData, processed: MutableMapping[str, Any]
+    ) -> None:
+        request = event_data.get("request") or {}
```

Comment:
I think you can put the empty dict as the default arg in `.get` like `event_data.get("request", {})`

Reply:
Good suggestion! Will do.